### ISSUE TITLE
False positive for "Receiver parameter is never used" on scoped Composable-Function with multiple emitters

### ISSUE THREAD

**Issue by florianklein-qm** [detekt/detekt#8748]
## Expected Behavior
UnusedReceiverParameter warning shouldn't appear.

## Observed Behavior
UnusedReceiverParameter is shown.

## Steps to Reproduce
```
@Composable
private fun RowScope.Content() {
    Text(text = "some content")
    Text(text = "some content")
}
```

=> Receiver parameter is never used

**Comment by atulgpt** [comment]
Hi @florianklein-qm is this a detekt rule? if yes,  Can you name the rule?
